client/js/components/faq/DpFaqSupport.vue

```diff
@@ -0,0 +1,121 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <div
+    id="scFaq"
+    class="c-support-wrapper space-inset-m">
+    <h2 class="font-normal">
+      {{ Translator.trans('support.heading') }}
+    </h2>
+    <section class="mb-5">
+      <p class="mb-5">
+        {{ Translator.trans('support.introduction') }}
+      </p>
+      <p>
+        Sie haben eine Frage zur Technik? Dann hilft Ihnen unser technischer Support gerne weiter.
+      </p>
+        <h3 class="font-size-h4 mt-5">
+          {{ Translator.trans('support.content') }}
+        </h3>
+      <ul class="grid lg:grid-cols-3 gap-3">
+        <li
+          v-for="contact in contacts"
+          class="space-inset-m h-48 c-support-singlecard">
+          <dp-faq-support-card>
+            <template v-slot:title>
+              <h2 class="font-semibold">
+                {{ contact.attributes.title }}
+              </h2>
+            </template>
+            <template v-slot:phonenumber>
+              <p class="mt-3 mb-0 inline-block font-semibold">
+                {{ contact.attributes.phoneNumber }}
+              </p>
+            </template>
+            <template v-slot:email>
+              <p class="mb-0 font-normal">
+                {{ contact.attributes.eMailAddress }}
+              </p>
+            </template>
+            <template v-slot:reachability>
+              <p class="mt-4 lg:mt-2 font-normal">
+                {{ (contact.attributes.text).replace(/<(?:"[^"]*"['"]*|'[^']*'['"]*|[^'">])+>/g, "") }}
+              </p>
+            </template>
+          </dp-faq-support-card>
+        </li>
+      </ul>
+    </section>
+    <h3 class="font-size-h4 font-semibold">
```

Comment:
same here for the `font-size-h4` and here a new `section` can be started

Reply:
done https://github.com/demos-europe/demosplan-core/pull/2139/commits/98c4a599f6ab47069611f3c0b89242c2728839e7